pkg/diagnose/diagnose.go

```diff
@@ -160,6 +163,47 @@ func (c *Collector) CollectPodLogs(corePodSelector labels.Selector) {
 	}
 }
 
+// collectSCCDescribe collect output of the "describe scc"
+func (c *Collector) collectSCCDescribe(scc *secv1.SecurityContextConstraints) {
+	cmd := exec.Command("kubectl", "describe", "scc", "-n", scc.Namespace, scc.Name)
+	// handle custom path for kubeconfig file,
+	// see --kubeconfig cli options
+	if len(c.kubeconfig) > 0 {
+		cmd.Env = append(cmd.Env, "KUBECONFIG=" + c.kubeconfig)
+	}
+
+	// open the out file for writing
+	fileName := c.folderName + "/" + scc.Name + "-scc-describe.txt"
+	outfile, err := os.Create(fileName)
+	if err != nil {
+		c.log.Printf(`❌ can not create file %v: %v`, fileName, err)
+		return
+	}
+	defer outfile.Close()
+	cmd.Stdout = outfile
+
+	// run kubectl describe
+	if err := cmd.Run(); err != nil {
+		c.log.Printf(`❌ can not describe scc %v namespace %v: %v`, scc.Name, scc.Namespace, err)
+	}
+}
+
+// CollectSCC collects the SCC 
+func (c *Collector) CollectSCC() {
+	for _, name := range []string{"noobaa", "noobaa-endpoint"} {
+		scc := &secv1.SecurityContextConstraints {
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      name,
+				Namespace: options.Namespace,
+			},
+		}
+		_, _, err := util.KubeGet(scc)
```

Comment:
Why not KubeCheckOptional?

Reply:
We only want to see that we can get is, chance it exists.
We don't want to check all the others (CRD, ect')

Reply:
We usually use KubeCheck, and I think it's much more readable just to do normal one if instead of handling the errors